components/lib/api.js

```diff
@@ -10,51 +11,63 @@ export const apiEndpoints = {
   COUNTRIES_LIST: '/api/_/countries',
   // Submissions
   SUBMISSION_LIST: '/api/_/url-submission/test-list',
-  SUBMISSION_ADD: '/api/v1/url-submission/add-url',
   SUBMISSION_UPDATE: '/api/v1/url-submission/update-url',
-  SUBMISSION_STATE: '/api/v1/url-submission/state',
-  SUBMISSION_CHANGES: '/api/v1/url-submission/changes',
   SUBMISSION_SUBMIT: '/api/v1/url-submission/submit'
 }
 
+const token = () => {
+  return typeof localStorage !== 'undefined' ? JSON.parse(localStorage.getItem('bearer'))?.token : ''
+}
+
 const axios = Axios.create({
   baseURL: process.env.NEXT_PUBLIC_OONI_API,
-  withCredentials: true
 })
 
+const refreshToken = async () => {
```

Comment:
I'm not sure if this is already done as a side-effect of the account_metadata call, but it might be worth explicitly having a periodic call to refreshToken so that we pre-emptively refresh it even if the user is not performing any actions on the page.